api/clients/v2/codecs/blob.go

```diff
@@ -93,6 +102,11 @@ func (b *Blob) toEncodedPayload(payloadForm codecs.PolynomialForm) (*encodedPayl
 		return nil, fmt.Errorf("invalid polynomial form")
 	}
 
+	maxPermissiblePayloadLength, err := codec.GetMaxPermissiblePayloadLength(b.blobLengthSymbols)
```

Comment:
have you been using `Get` prefixes everywhere? I tend to prefer not having those, but as long as we're consistent everywhere.

Reply:
I tend to use `Get`, but don't have a strong preference. I will start leaving it off in the future (though universal consistency is gonna prove elusive, since I think we have instances of both naming strategies throughout)